packages/app/src/app/pages/Sandbox/Editor/index.tsx

```diff
@@ -94,7 +94,7 @@ const ContentSplit: React.FC = () => {
               right: 0,
               bottom: bottomOffset,
               height: statusBar ? 'auto' : 'calc(100% - 3.5rem)',
-              zIndex: 999999,
+              zIndex: 9,
```

Comment:
@CompuIves @christianalfoni I'm sure this breaks something, any tips on where should I start looking?